docs/pages/docs/editor-basics/setup.mdx

```diff
@@ -0,0 +1,135 @@
+---
+description: Learn how to setup your BlockNote editor using the `useCreateBlockNote` hook and the ``BlockNoteView` component.
+---
+
+import { Example } from "@/components/example";
+import { Callout } from "nextra/components";
+
+# Editor Setup
```

Comment:
Did you remove the docs for `BlockNoteEditor.create`? Isn't that also relevant for editor setup, or do you think it's too niche of a use case?